### ISSUE TITLE
bug: "Search" input value not read out loud on screen readers

### ISSUE THREAD

**Issue by JCC-DFO** [cds-snc/gcds-components#1190]
### Prerequisites | Prérequis

- [x] I have read the [Contributing Guidelines](https://github.com/cds-snc/gcds-components/blob/main/CONTRIBUTING.md). | J’ai lu les [Lignes directrices sur les contributions](https://github.com/cds-snc/gcds-components/blob/main/CONTRIBUTING.md).
- [x] I agree to follow the [Code of Conduct](https://github.com/cds-snc/gcds-components/blob/main/CODE_OF_CONDUCT.md). | J’accepte de me conformer au [Code de conduite](https://github.com/cds-snc/gcds-components/blob/main/CODE_OF_CONDUCT.md).
- [x] I have searched for [existing issues](https://github.com/cds-snc/gcds-components/issues) that already report this problem, without success. | J’ai vérifié parmi [les tickets (issues) existants](https://github.com/cds-snc/gcds-components/issues) que ce sujet n’avait pas déjà été soumis.

### GC Design System Components Package and Version | Paquet et version des composants de Système de design GC

gcds-components @latest

### Current Behavior | Comportement observé

When using a screen reader, when tabbing onto the "Search" component, the value of the input field (text written) is not read out loud.

### Expected Behavior | Comportement attendu

The value of the input field (text written) should be read out loud by the screen reader.

### System Info | Information sur le système

```shell

```

### Steps to Reproduce | Étapes pour reproduire le bogue

Render a <GCDS-Search /> component in a web page. Use a tool like Windows Screen Reader/Narrator, and focus onto the Search component.

**Comment by adorayi** [comment]
Reached out and the team and confirmed this can be closed; no longer an issue.
